src/main/java/webserver/RequestHandler.java

```diff
@@ -24,16 +32,61 @@ public void run() {
 
         try (InputStream in = connection.getInputStream(); OutputStream out = connection.getOutputStream()) {
             // TODO 사용자 요청에 대한 처리는 이 곳에 구현하면 된다.
+            BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(in));
+
+            HttpRequest httpRequest = HttpRequest.of(bufferedReader);
+
             DataOutputStream dos = new DataOutputStream(out);
-            byte[] body = "Hello World".getBytes();
-            response200Header(dos, body.length);
+
+            byte[] body = getBodyFromRequest(httpRequest);
+
+            response200Header(dos, body.length, httpRequest.getHttpRequestPath().getPath());
             responseBody(dos, body);
+        } catch (Exception e) {
+            logger.error(e.getMessage());
+        }
+    }
+
+    private static byte[] getBodyFromRequest(HttpRequest httpRequest) throws IOException, URISyntaxException {
+
+        if (httpRequest.getHttpRequestPath().isEndsWith(".html")) {
```

Comment:
httpRequest 객체에게 바로 요청해볼 수 있겠네요.  
(httpRequest는 HttpRequestPath의 isEndWith 메서드를 그대로 포워딩해주게되겠네요. 🙂)

```suggestion
        if (httpRequest.isEndsWith(".html")) {
```